core/connection.go

```diff
@@ -261,6 +264,9 @@ func mustGetAddress(chain interface {
 
 func checkConnectionFinality(src, dst *ProvableChain, srcConnection, dstConnection *conntypes.ConnectionEnd) (bool, error) {
 	logger := GetConnectionPairLogger(src, dst)
+	if src.Path().ConnectionID == "" && dst.Path().ConnectionID == "" {
+		return true, nil
+	}
```

Comment:
Do we need to specially handle this case?
We can still normally handle the case where the src connID is empty but the dst connID is NOT empty, right?

Reply:
Removed